src/main/java/Command.java

```diff
@@ -0,0 +1,3 @@
+public abstract class Command {
+    public abstract void execute(Ui ui, Storage storage);
```

Comment:
If I am not wrong, I think abstract methods should require JavaDoc comments as well?